internal/services/securitycenter/security_center_setting_resource.go

```diff
@@ -46,6 +46,10 @@ func resourceSecurityCenterSetting() *pluginsdk.Resource {
 				ValidateFunc: validation.StringInSlice([]string{
 					"MCAS",
 					"WDATP",
+					"Sentinel",
+
+					// TODO 4.0: Remove this per the swagger definition:
+					//           https://github.com/Azure/azure-rest-api-specs/blob/b52464f520b77222ac8b0bdeb80a030c0fdf5b1b/specification/security/resource-manager/Microsoft.Security/stable/2021-06-01/settings.json#L285
```

Comment:
Can we please put this behind the `features.FourPointOhBeta()` flag instead of a comment

Reply:
Sure, updated!